Makefile

```diff
@@ -20,7 +20,7 @@ GOLANGCI_VERSION   = 1.52.0
 GOTESTSUM_VERSION ?= 1.9.0
 
 PROTOC_VERSION             = 21.12
-PROTOC_GEN_GO_VERSION      = 1.28.1
+PROTOC_GEN_GO_VERSION      = 1.30.0
```

Comment:
The previous generation of our protobuf files used 1.30.0 but the make file was not updated too match